### PR TITLE
Docs: Replace MySQL with SQL in wp-config-sample.php

### PR DESCRIPTION
**Pull request by audrasjb**
Trac ticket: https://core.trac.wordpress.org/ticket/54610

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: wp-config-sample.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
  *
  * This file contains the following configurations:
  *
- * * MySQL settings
+ * * SQL settings
```
Comment (hellofromtonya) [inline wp-config-sample.php]: @audrasjb How about `Database` instead? Why? These settings are not specifically sql commands, but rather constants to define the database settings.

```suggestion
 * * Database settings
```

File: wp-config-sample.php [hunk 2]
```diff
@@ -18,17 +18,17 @@
  * @package WordPress
  */
 
-// ** MySQL settings - You can get this info from your web host ** //
+// ** SQL settings - You can get this info from your web host ** //
```
Comment (hellofromtonya) [inline wp-config-sample.php]:
```suggestion
// ** Database settings - You can get this info from your web host ** //
```

File: wp-config-sample.php [hunk 3]
```diff
@@ -18,17 +18,17 @@
  * @package WordPress
  */
 
-// ** MySQL settings - You can get this info from your web host ** //
+// ** SQL settings - You can get this info from your web host ** //
 /** The name of the database for WordPress */
 define( 'DB_NAME', 'database_name_here' );
 
-/** MySQL database username */
+/** SQL database username */
```
Comment (hellofromtonya) [inline wp-config-sample.php]:
```suggestion
/** Database database username */
```

File: wp-config-sample.php [hunk 4]
```diff
@@ -18,17 +18,17 @@
...
+// ** SQL settings - You can get this info from your web host ** //
 /** The name of the database for WordPress */
 define( 'DB_NAME', 'database_name_here' );
 
-/** MySQL database username */
+/** SQL database username */
 define( 'DB_USER', 'username_here' );
 
-/** MySQL database password */
+/** SQL database password */
```
Comment (hellofromtonya) [inline wp-config-sample.php]:
```suggestion
/** Database database password */
```

File: wp-config-sample.php [hunk 5]
```diff
@@ -18,17 +18,17 @@
...
 define( 'DB_PASSWORD', 'password_here' );
 
-/** MySQL hostname */
+/** SQL hostname */
```
Comment (hellofromtonya) [inline wp-config-sample.php]:
```suggestion
/** Database hostname */
```

**Review by hellofromtonya — APPROVED** [review]
Marking for commit 👍

**Comment by audrasjb** [comment]
Committed in https://core.trac.wordpress.org/changeset/52367

**Pull request closed without merging**